change GNU compliant Makefile PREFIX variable to Debian package compliant
--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,7 @@
 # determination of the actual installation directories.
 # Suggested values are "/usr/local", "/usr", "/home/user/programs"
 #
-PREFIX = /usr/local
+PREFIX = /usr
 
 #
 # Certain platforms do not support long options (command line options).
